src/tools/sfdk/sdkmanager.cpp

```diff
@@ -1353,21 +1353,18 @@ bool SdkManager::mapEnginePaths(QString *program, QStringList *arguments, QStrin
         QString hostPath;
         QString enginePath;
         Qt::CaseSensitivity cs;
-    } const mappings[] = {
-#if Q_CC_GNU <= 504 // Let's check if it is still needed with GCC > 5.4
-        {cleanSharedTarget + QLatin1String("/"), QLatin1String("/"), Qt::CaseSensitive},
-        {cleanSharedTarget, QLatin1String("/"), Qt::CaseSensitive},
-        {cleanSharedHome, QLatin1String(Constants::BUILD_ENGINE_SHARED_HOME_MOUNT_POINT), Qt::CaseSensitive},
-        {cleanSharedSrc, QLatin1String(Constants::BUILD_ENGINE_SHARED_SRC_MOUNT_POINT), caseInsensitiveOnWindows}
-#else
```

Comment:
It would be nice to have the <=504 case removal in a separate commit :)

Reply:
Moved to separate commit